src/jsonschema_wtforms/field.py

```diff
@@ -247,14 +249,22 @@ def from_json_field(
 
         requirements = params.get('required', [])
         fields = {}
+        definitions = params.get('definitions', {})
         for property_name, definition in properties.items():
             if property_name not in include:
                 continue
-            if (type := definition.get('type', None)) is not None:
-                field = converter.lookup(type)
+            if ref := definition.get('$ref'):
+                if not definitions:
+                    raise NotImplementedError('Missing definitions.')
```

Comment:
Line 178, we have an assertion for  this, here a NotImplementedError. We might want to harmonize it.